### PR TITLE
Test jenkinsci/job-config-history-plugin#338

### PR DESCRIPTION
**Pull request by shlomomdahan**
https://github.com/jenkinsci/job-config-history-plugin/pull/338

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -496,6 +496,10 @@ and
           </systemPropertyVariables>
           <!-- SUREFIRE-1588 workaround; too late for systemProperties: -->
           <argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
+          <environmentVariables>
+            <!-- Use local version of the plugin over the released one ATH would otherwise use -->
+            <LOCAL_JARS>target/job-config-history-plugin.hpi</LOCAL_JARS>
```
Comment (basil) [inline pom.xml:501]: The artifact ID is `jobConfigHistory`, so this should be

```suggestion
            <LOCAL_JARS>target/jobConfigHistory.hpi</LOCAL_JARS>
```

**Comment by basil** [comment]
https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-1779/2/ passed

**Comment by shlomomdahan** [comment]
@basil 

is the below expected?

Screenshot 2024-10-10 at 1 56 45 PM

**Comment by basil** [comment]
Yes, see https://github.com/jenkinsci/acceptance-test-harness/issues/1052.

**Pull request closed without merging**